### ISSUE TITLE
error in addition_rnn example

### ISSUE THREAD

**Issue by wolet** [fchollet/keras#782]
I think on [line 151](https://github.com/fchollet/keras/blob/master/examples/addition_rnn.py#L151) should be like this:

      model.fit(X_train, y_train, batch_size=BATCH_SIZE, nb_epoch=1, validation_data=(X_val, y_val), show_accuracy=True)

because X already contains X_val, y_val.

**Comment by fchollet** [comment]
Yes, you're right. Would you like to submit a PR?

**Comment by Smerity** [comment]
Can't believe I screwed that up. Oops oops. Re-running the numbers now for the documentation at the top of the code. @wolet, if you'd like to commit the fix I can send you the updated numbers, else I can just push the commit through myself.

Thanks for the eagle eyes!

**Comment by Smerity** [comment]
Fix pushed as part of https://github.com/fchollet/keras/commit/2bd4c295d6cae124f2cffce0e7343dc2c18d7eee#diff-c907b2d87405fcbcf1de291078dcac52L151

Only bit left not fixed is that, from my tests, it now takes ~100 epochs instead of ~55 to get 99% accuracy for two digit inverted + 5k training examples.
